### ISSUE TITLE
Media Player has no app icon

### ISSUE THREAD

**Issue by electr1fy0** [BobbyESP/Jetispot#12]
### Describe the bug

(Android 13)
Notice the blank circle where the icon should be.

### To Reproduce

Just play

### Error reports & Screenshots

![Screenshot_20230109-180448](https://user-images.githubusercontent.com/87616300/211309525-b5460455-757c-47c5-b0da-c5ae38842b65.png)

### Device info

Pixel 5

**Comment by BobbyESP** [comment]
Yeah, Media Player is so bugged, that's not the only problem with it that the app has. I'll investigate

**Comment by electr1fy0** [comment]
Unfortunately, after the report. I faced a lot of slowdowns and crashes as well in my limited usage.
I might give it another go tonight.

**Comment by BobbyESP** [comment]
Are you a non-premium account user?

**Comment by electr1fy0** [comment]
Yes, I haven't renewed my Premium yet.

**Comment by BobbyESP** [comment]
Okay, that is probably the problem. The app is not supposed to be for non-premium users but they can login on it and have unlimited skips and no ads

**Comment by electr1fy0** [comment]
I renewed my Premium and the first track I played ended up in a crash. 🫠

**Comment by BobbyESP** [comment]
Sorry for the inconveniences. Jetispot needs a big update and fixes. Since I'm doing Spowlo (a music songs downloader), I'm focused on it, but when I have time I will continue with Jetispot. This bug happens sometimes when clicking two songs at the same time for example.
